web/src/main/java/org/mskcc/cbio/oncokb/api/pub/v1/AnnotationsApiController.java

```diff
@@ -557,23 +612,33 @@ private IndicatorQueryResp getIndicatorQueryFromGenomicLocation(
             null,
             false,
             evidenceTypes,
-                false
+            false
         );
     }
 
+    private Map<String, Alteration> getAlterationsFromGenomeNexus(GNVariantAnnotationType type, ReferenceGenome referenceGenome, List<String> queries) throws ApiException, org.genome_nexus.ApiException {
+        List<String> queriesToGN = new ArrayList<>();
+
+        for (String query : queries) {
```

Comment:
Should we filter out queries that are empty strings at this step. This looks like the entry point to get GN annotation.

Reply:
`genomicLocationShouldBeAnnotated` will filter out the empty query.